compute/elb.tf

```diff
@@ -0,0 +1,29 @@
+resource "aws_elb" "load-balancer" {
+  name               = "load-balancer"
+  availability_zones = ["us-east-1a", "us-east-1b", "us-east-1c"]
```

Comment:
You can use this datasource instead:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/availability_zones